hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -334,9 +335,11 @@ async def jobs(self, q=None):
             if last_job_id is None:
                 break
 
-    async def status(self):
-        resp = await self._client._get(f'/api/v1alpha/batches/{self.id}')
-        return await resp.json()
+    async def status(self, refresh=False):
```

Comment:
You probably need to add these to hailtop/batch_client/client.py as well.

Reply:
You're right, I need to at least add the caching initialization to `client.Batch.__init__`. Will do.

Reply:
Hold on. Do I though? The client `Batch` class is only ever constructed by `_from_async_batch` and it seems like we don't want consumers of this API actually calling `client.Batch.__init__` or `aioclient.Batch.__init__`.

Reply:
You meant the `refresh` parameter, right. Done now.